hooks/open-telemetry/README.md

```diff
@@ -95,3 +110,19 @@ OpenFeatureAPI api = OpenFeatureAPI.getInstance();
 // Register MetricsHook with custom dimensions
 api.addHooks(new MetricsHook(openTelemetry, customDimensions));
 ```
+
+Alternatively, you can wrtie your own extraction logic against [flag evaluation metadata](https://github.com/open-feature/spec/blob/main/specification/types.md#flag-metadata) by providing a callback to `TracesHookOptions.dimensionExtractor`.
```

Comment:
nit: here we say dimensionExtractor but the code down uses attributeSetter

Reply:
good catch, updated with the latest commit